### PR TITLE
added documentation and design

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- added documentation and design
- added some design elements to the overview

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [0.0.0.0:8024](http://0.0.0.0:8024/)
- see that it looks a bit like the [copy doc](https://docs.google.com/document/d/1ZmYO1DIO9bUlaYvThUWF5vSSp5OVsmBoa8nRu15usm0/edit#)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,33 +1,46 @@
 ---
 layout: default
-desc: Netplan.io
+desc: netplan.io
```
Comment (anthonydillon) [inline index.html]: Is it uppercase or lowercase? If lowercase we should be constraint even at the beginning of a sentence.

File: index.html [hunk 2]
```diff
@@ -1,33 +1,46 @@
...
       </p>
       <p>Netplan is a utility for easily configuring networking on a linux system. You simply create a YAML description of the required network interfaces and what each should be configured to do. From this description Netplan will generate all the necessary configuration for your chosen renderer tool.</p>
+    </div>
+    <div class="col-4 u-align--center u-hidden--small">
+      <img src="https://assets.ubuntu.com/v1/0fbee9c4-pictogram_internet01.svg?w=200" width="200" alt="" />
+    </div>
+  </div>
+</div>
+
+<div id="quick-start" class="p-strip--light is-bordered is-deep">
```
Comment (anthonydillon) [inline index.html]: id="how-does-it-work"

File: index.html [hunk 3]
```diff
@@ -1,33 +1,46 @@
...
+desc: netplan.io
 sitemap:
     priority: 1.0
     changefreq: 'monthly'
     lastmod: 2017-01-13T17:20:30+00:00
 ---
 
-<div id="main-content" class="p-strip is-bordered is-deep">
-  <div class="row">
+<div id="main-content" class="p-strip--hero is-bordered is-deep">
```
Comment (anthonydillon) [inline index.html]: Is there such a thing as `p-strip--hero`?

File: index.html [hunk 4]
```diff
@@ -1,33 +1,46 @@
...
+        <li class="p-list__item is-ticked">
+          <a href="https://wiki.ubuntu.com/NetworkManager" class="p-link--external">NetworkManager</a>
+        </li>
+        <li class="p-list__item is-ticked">
+          <a href="http://manpages.ubuntu.com/manpages/bionic/man5/systemd.network.5.html" class="p-link--external">Systemd-networkd</a>
+        </li>
+      </ul>
     </div>
   </div>
-</div>
```
Comment (anthonydillon) [inline index.html]: You need to choose the strip above.

File: index.html [hunk 5]
```diff
@@ -57,16 +70,6 @@ <h3>Commands</h3>
...
-<div class="p-strip--accent is-bordered is-shallow">
-  <div class="row">
-    <div class="col-8">
-      <h3>Commercial support</h3>
-      <p>For commerical support please contact us</p>
-      <p><a href="/contact-us" class="p-button">Contact support</a></p>
-    </div>
-  </div>
-</div>
-
```
Comment (anthonydillon) [inline index.html]: No commercial support?

File: index.html (hunk 2 above)
Comment (pmahnke) [reply index.html]: done.

File: index.html (hunk 1 above)
Comment (pmahnke) [reply index.html]: I think the url is lower and the title is upper? you?

File: index.html (hunk 3 above)
Comment (pmahnke) [reply index.html]: removed

File: index.html (hunk 5 above)
Comment (pmahnke) [reply index.html]: I think we need to get help from Thibaut first.

File: design.html [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+        <li>No persistent generated config, only original YAML config</li>
+        <li>Default policy applies with no config file present</li>
+        <li>Parser supports multiple config files to allow applications (libvirt, lxd) to package up expected network config (virbr0, lxdbr0), or to change the global default policy to use <a href="https://wiki.ubuntu.com/NetworkManager">NetworkManager</a>          for everything.</li>
+        <li>Retains the flexibility to change backends/policy later or adjust to "apt purge network-manager" as generated configuration is ephemeral</li>
+      </ul>
+
+      <h2>Requirements</h2>
+
+      <ul>
+        <li>Initial configuration of network on a host from an oracle: MaaS, Cloud, Installer (Subiquity/Ubiquity), WebDM</li>
```
[the remark this answers is not in the document]
Comment (caldav) [reply design.html]: It's called snapweb now (not sure about capitalisation) -- and work on it has just resumed.

File: index.html (hunk 4 above)
Comment (pmahnke) [reply index.html]: I don't see that

**Comment by pmahnke** [comment]
@anthonydillon I fixed a lot of these in the next PR... by mistake... I am closing this.

**Pull request closed without merging**